olive/engine/packaging/packaging_generator.py

```diff
@@ -170,10 +172,19 @@ def _upload_to_azureml_data(
     )
 
 
-def _get_model_info(node: "FootprintNode", model_rank: int):
+def _get_model_info(
```

Comment:
we might need add UT to verify the path is correct.

Reply:
I'll refactor the whole tests of this generate packaging api and add tests in a follow-up PR.

Reply:
could we do the UT refactoring in this PR? if not, how could this change behavior correctly?

Reply:
Before sending this PR, I verified this manually with this change.

Reply:
better to unzip the package file in UT to verify the path to make sure the PR is complete. I suppose it is just one line change.

Reply:
> I'll refactor the whole tests of this generate packaging api and add tests in a follow-up PR.

Before next refactoring PR, we need add test for the fix in this PR.

Reply:
> Before sending this PR, I verified this manually with this change.

from the PR, it seems the path is still wrong?

Reply:
![image](https://github.com/microsoft/Olive/assets/817030/5c67166d-fd6e-46d0-9d4a-726ea3ae36a7)
The model path is still absolute path?